Pipfile

```diff
@@ -21,9 +21,11 @@ pytest-cov = "*"
 functions-framework = "*"
 python-dotenv = "*"
 types-requests = "*"
+wrapt = "*"
+dill = "*"
```

Comment:
Not sure why we need this?. How come some repos don't need it?. What does the error output look like without this?

Reply:
https://github.com/ONSdigital/eq-submission-confirmation-consumer/actions/runs/5726110288/job/15515900979 heres a link to the error. I haven't been able to find it referenced anywhere in the repo other than in the pipfile.lock. I think that maybe its no longer included in the python release and it is used somewhere by one of the other dependencies. I did some research but didn't find much, adding them in was the only way I managed to find of fixing the error.

Reply:
But I am not able to recreate this error locally?. I remove these from pipfile and ensure they are not install via pip and it all seems to work fine. 🤔

Reply:
Have you tried using `setup-python@v4` in actions?

Reply:
Looks like dill is already a dependency of pylint, so is astroid though pinned.

https://github.com/pylint-dev/pylint/blob/main/pyproject.toml#L38

And astroid remove `wrap` in v2 https://github.com/pylint-dev/astroid/commit/cff437ab3bfed3ec04958765ce4bd560a22c9220